app/javascript/readingList/components/__tests__/ItemListItem.test.jsx

```diff
@@ -2,7 +2,6 @@ import { h } from 'preact';
 import { render } from '@testing-library/preact';
 import { axe } from 'jest-axe';
 import { ItemListItem } from '../ItemListItem';
-import '../../../../assets/javascripts/lib/xss';
```

Comment:
This is not part of the fix, but I saw it while investigating the bug. This line is not necessary as the xss lib is already loaded via

https://github.com/forem/forem/blob/0d6db30c7e64c96416a78a779d0c67e868628810/testSetup.js#L3